person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @age = age
+    @rentals = []
```

Comment:
- [ ] the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Kindly check out the example below ⬇️  Please implement it in your code to make it completed 👍
```
 def add_rental(book, date)
    Rental.new(date, book, self)
  end

```

Reply:
Thanks for your insights @codecaiine, I will implement it right now.